components/View/Tooltip/index.js

```diff
@@ -125,6 +128,10 @@ export default class Tooltip extends React.PureComponent {
         };
         const isTooltipNode = typeof tooltip === 'object';
 
+        if (isFalsyString(tooltip)) {
```

Comment:
There is a better way to do this.

At line 138, instead of `showTooltip` we can have `showTooltip j&& isTruthyString(tooltip)`